packages/distribution/addon/routes/index.js

```diff
@@ -1,41 +1,28 @@
 import Route from "@ember/routing/route";
 import { inject as service } from "@ember/service";
-import { queryManager } from "ember-apollo-client";
-
-import config from "@projectcaluma/ember-distribution/config";
-import inquiryNavigationQuery from "@projectcaluma/ember-distribution/gql/queries/inquiry-navigation.graphql";
-import uniqueByGroups from "@projectcaluma/ember-distribution/utils/unique-by-groups";
 
 export default class IndexRoute extends Route {
+  @service("-scheduler") scheduler;
+  @service distribution;
   @service router;
-  @service calumaOptions;
 
-  @config config;
+  async redirect() {
+    // trigger resource
+    this.distribution.navigation.value;
```

Comment:
Why do we have to trigger the resource in this way?

Reply:
How else?